ci/zinc/linux_test.sh

```diff
@@ -45,7 +45,7 @@ cd $WORKSPACE
 
 # Look for non-conforming code formatting.
 # Formatting errors can be fixed by running `zig fmt` on the files printed here.
-$ZIG fmt --check .
+$ZIG fmt --check . --exclude test/compile_errors/
```

Comment:
Why is the formatting check disabled for test cases in the CI? If there problems/bugs, then they should be linked here.

Reply:
This is because the tested compile errors are sometimes parsing errors. Some of these test files fail when run with zig fmt

Additionally, the existing expected error messages reference the line/column numbers of the sources. Formatting them would require a lot of work to update the expected error messages